dot/node.go

```diff
@@ -299,10 +293,4 @@ func (n *Node) Stop() {
 	// stop all node services
 	n.Services.StopAll()
 
-	// close node stop channel if not already closed
-	if n.stop != nil {
-		close(n.stop)
-	}
```

Comment:
this can probably stay? it's only being closed in one spot and read from in one spot

Reply:
does this stop need to be removed?

Reply:
hmmm, yes, not really. I restored it and it seems to work properly when using range loop. :+1:

Reply:
Looks like @noot requested this not be removed and you mention restoring it - could you clarify where/how you restored it or is this no longer an issue?

Reply:
I restored it with a wait group. Please check again.